src/pip/_internal/models/wheel.py

```diff
@@ -58,7 +58,10 @@ def support_index_min(self, tags: List[Tag]) -> int:
         :raises ValueError: If none of the wheel's file tags match one of
             the supported tags.
         """
-        return min(tags.index(tag) for tag in self.file_tags if tag in tags)
+        try:
+            return next(i for i, t in enumerate(tags) if t in self.file_tags)
+        except StopIteration:
+            raise ValueError()
```

Comment:
Is this equivalent and perhaps easier to read ?

```suggestion
        for i, t in enumerate(tags):
            if t in self.file_tags:
                return i
        raise ValueError()
```

Reply:
Hehe, see https://github.com/pypa/pip/pull/11128#discussion_r876464258

Reply:
Aaah. A matter of taste, then, I guess :)